expan/core/experiment.py

```diff
@@ -351,4 +351,4 @@ def chi_square_test_result_and_statistics(self, variant_column, weights, min_cou
         # Compute chi-square and p-value statistics
         chi_square_val, p_val = statx.chi_square(observed_freqs.sort_index(), expected_freqs.sort_index())
 
-        return p_val >= alpha, p_val, chi_square_val
+        return p_val >= alpha, p_val, chi_square_val, observed_freqs, expected_freqs
```

Comment:
Is this going to be used in ADS?

Reply:
I'm still periodically getting surprised why you remind about ADS here.
Expan shouldn't care, it is a lib.

Reply:
yes, in ADS we want to log the percentage difference between observed and expected counts 
along with the chi-square results to have more information about the split for the experiment. I don't want to duplicate the counts computation in ADS since we already have them here.

Reply:
@igusher I know, I am asking just for my understanding. And Gute Bessrung btw! ;)